socket.go

```diff
@@ -8,16 +8,30 @@ import (
 )
 
 // parseSockAddr resolves given addr to unix.Sockaddr
-func parseSockAddr(addr string) (unix.Sockaddr, error) {
+func parseSockAddr(addr string) (unix.Sockaddr, int, error) {
```

Comment:
A word for the family`int`?

Reply:
It use word in the definition of [unix.Socket](https://cs.opensource.google/go/x/sys/+/f75cf1ee:unix/syscall_unix.go;l=484), so I keep it.

Reply:
fixed